AsyncDisplayKit/ASTextNode.h

```diff
@@ -12,6 +12,14 @@
 @protocol ASTextNodeDelegate;
 
 /**
+ * Alignment
+ */
+typedef NS_ENUM(NSUInteger, ASTextNodeVerticalAlignment) {
```

Comment:
Let's actually make this type ASTextNodeAlignment.  That way, we can use the same enum for horizontal alignment later.  The main reason I'd prefer this is because it will make the autocomplete much easier to find; ASTextNodeAl [tab] would work, whereas guessing / remembering the necessity of Ver[tab] is tougher.